### DISCUSSION TITLE
Add <Table> support for custom component

### DISCUSSION THREAD

**Discussion by R0bk** [gpbl/react-day-picker#1420] [Ideas]
Hey all,

I was wondering if it would be possible to add the <Table> component in as a custom component so that we can then add more rows to the month table if we want to (I've got an application where I'd like a row to appear under the date that the user clicks but this currently isn't possible).

Here's a quick poc repo if this has any support https://github.com/R0bk/react-day-picker.

**Comment by gpbl** [comment]
@R0bk would you mind to clean up [the diff](https://github.com/gpbl/react-day-picker/compare/master...R0bk:master) from the `build`? I think however more rows in the table would break accessibility. 

> a row to appear under the date that the user clicks 

An example of the rendering would be great! Moving to Discussions...
